src/Providers/Stripe/Payment/Service.php

```diff
@@ -51,15 +51,13 @@ public function create(
     ): Payment {
         switch ($token->type) {
             case StripeToken::CUSTOMER:
-                $customer = $this->customers()->find($token->value);
-
-                $cards = $this->customers()->getCards($customer);
+                $cards = $this->customers()->getCards($token->value);
                 $card = $cards[0];
```

Comment:
`$this->customers()->getCardsForCustomer()` would be more accurate for the current codebase.

Would be nicer to read `getCards()->first()`.
Or `getFirstCard()` if you don't want to bring Collections in.

Reply:
This is a workaround the opencharge web app where we are charging customers directly. We shouldn't do that because we should handle multiple cards in the first place so adding a new method based on a lack of functionality seems irrelevant IMO. But I get your point.